examples/Model.elm

```diff
@@ -313,8 +333,8 @@ fetchRecord recordId =
         |> Kinto.send FetchRecordResponse
 
 
-fetchRecordList : Sort -> Cmd Msg
-fetchRecordList sort =
+fetchRecordList : Sort -> Maybe Int -> Cmd Msg
+fetchRecordList sort limit =
```

Comment:
I feel like this function could benefit from accepting a `params` record holding `sort`, `limit`, `filters` and friends (especially as these are stored in the model already), but that may just be overkill for this example app.